lib/formatters.js

```diff
@@ -73,9 +74,35 @@ const serialize = (obj) => {
         .join(',');
 };
 
+function flatten(data, prefix = 'data') {
```

Comment:
Where are your tests for this function? Or do the existing tests adequately cover the objects you intend to flatten?

Also, is `flatten` too generic of a name? Does this flatten _any_ object? Or just certain types of objects?

Reply:
Oh, that's a good point. I should add tests for that!

I don't think `flatten` is too generic. It does flatten _any_ object